packages/block-editor/src/hooks/color.js

```diff
@@ -180,9 +180,13 @@ const getLinkColorFromAttributeValue = ( colors, value ) => {
  */
 export function ColorEdit( props ) {
 	const { name: blockName, attributes } = props;
-	const { colors, gradients } = useSelect( ( select ) => {
-		return select( 'core/block-editor' ).getSettings();
-	}, [] );
+	const { colors, gradients, __experimentalEnableLinkColor } = useSelect(
```

Comment:
do you think it's possible to use the `useFeature` hook introduced by @gziolo or is it only for theme.json for now?

Reply:
yeah, it only read from theme.json and not yet from `add_theme_support`